prow/crier/controller.go

```diff
@@ -159,14 +163,14 @@ func (r *reconciler) reconcile(ctx context.Context, log *logrus.Entry, req recon
 	for _, pjob := range pjs {
 		if err := criercommonlib.UpdateReportStateWithRetries(ctx, pjob, log, r.pjclientset, r.reporter.GetName()); err != nil {
 			log.WithError(err).Error("Failed to update report state on prowjob")
-			// The error above is alreay logged, so it would be duplicated
+			// The error above is already logged, so it would be duplicated
```

Comment:
Please do not mix unrelated typo fixes into a single commit (you can do it in a separate commit).

Reply:
Reverted the change.

Reply:
I would have preferred if you still fixed the (obviously wrong) typo as a separate commit. But as it is not the main focus of this PR, I'm OK with dropping it.

Reply:
Pushed a new commit with the typo fix.